packages/webclient/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "azure-actions-webclient",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Authorize to azure and make rest calls",
   "keywords": [
     "webclient",
     "actions"
   ],
   "scripts": {
     "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "prepare": "npm run build"
+    "copypackage": "copy package.json lib",
+    "dist": "npm run build && npm run copypackage && cd lib && npm publish"
```

Comment:
Please add comment in package.json on how to publish for future contributors

Reply:
Since comments can not be added in json, please document this in readme for this package